packages/interflex/src/__tests__/translate.test.ts

```diff
@@ -0,0 +1,115 @@
+import t, { injectDictionaries, generateDefaultDictionary } from '../translate';
+
+describe('translate function', () => {
+  beforeAll(() => {
+    injectDictionaries({
+      en: generateDefaultDictionary({
+        simple: 'Welcome!',
+        interpolate: 'Welcome, {name}.',
+        complex: 'Welcome! Please click <Link>here</Link>.',
+        complexInterpolate: 'Welcome, <foo>{name}</foo>.',
+        nextedComplexInterpolate:
```

Comment:
Is this catchy word intentional? I kinda like that `nexted`, sounds like `next level nested` to me. 😂

Reply:
LMAO! No, that's a typo 🤣